odd-platform-ui/src/components/DataEntityDetails/Overview/OverviewDataQualityReport/OverviewDataQualityReportStyles.ts

```diff
@@ -1,122 +1,55 @@
-      [`&.${DataQualityTestRunStatusEnum.UNKNOWN}`]: {
-        color: theme.palette.runStatus.UNKNOWN,
-      },
-    },
-    barsContainer: {
-      marginTop: theme.spacing(0.5),
-      marginBottom: theme.spacing(0.5),
-    },
-    bar: ({
-      datasetQualityTestReport,
-    }: {
-      datasetQualityTestReport?: DataSetTestReport;
-    }) => {
-      // Stretching bars if total width less than 50%
-      const succRelation =
-        (datasetQualityTestReport?.successTotal || 0) /
-        (datasetQualityTestReport?.total || 1);
-      const otherStatusesAdjustment =
-        200 / (Math.round(1 - succRelation) + 1);
-      return {
-        height: '8px',
-        width: '100%',
-        [`&.${DataQualityTestRunStatusEnum.SUCCESS}`]: {
-          backgroundColor: theme.palette.runStatus.SUCCESS,
-          maxWidth: `${
-            (succRelation * 200) / (Math.round(succRelation) + 1)
-          }%`,
-        },
-        [`&.${DataQualityTestRunStatusEnum.FAILED}`]: {
-          backgroundColor: theme.palette.runStatus.FAILED,
-          maxWidth: `${
-            ((datasetQualityTestReport?.failedTotal || 0) /
-              (datasetQualityTestReport?.total || 1)) *
-            otherStatusesAdjustment
-          }%`,
-        },
-        [`&.${DataQualityTestRunStatusEnum.BROKEN}`]: {
-          backgroundColor: theme.palette.runStatus.BROKEN,
-          maxWidth: `${
-            ((datasetQualityTestReport?.brokenTotal || 0) /
-              (datasetQualityTestReport?.total || 1)) *
-            otherStatusesAdjustment
-          }%`,
-        },
-        [`&.${DataQualityTestRunStatusEnum.SKIPPED}`]: {
-          backgroundColor: theme.palette.runStatus.SKIPPED,
-          maxWidth: `${
-            ((datasetQualityTestReport?.skippedTotal || 0) /
-              (datasetQualityTestReport?.total || 1)) *
-            otherStatusesAdjustment
-          }%`,
-        },
-        [`&.${DataQualityTestRunStatusEnum.ABORTED}`]: {
-          backgroundColor: theme.palette.runStatus.ABORTED,
-          maxWidth: `${
-            ((datasetQualityTestReport?.abortedTotal || 0) /
-              (datasetQualityTestReport?.total || 1)) *
-            otherStatusesAdjustment
-          }%`,
-        },
-        [`&.${DataQualityTestRunStatusEnum.UNKNOWN}`]: {
-          backgroundColor: theme.palette.runStatus.UNKNOWN,
-          maxWidth: `${
-            ((datasetQualityTestReport?.unknownTotal || 0) /
-              (datasetQualityTestReport?.total || 1)) *
-            otherStatusesAdjustment
-          }%`,
-        },
-      };
-    },
-  });
-
-export type StylesType = WithStyles<typeof styles>;
+export const Container = styled('div')(() => ({
+  width: '100%',
+  display: 'flex',
+  justifyContent: 'stretch',
+  alignItems: 'stretch',
+  justifySelf: 'stretch',
+  flexGrow: 1,
+}));
+export const CountLabel = styled(Typography, {
+  shouldForwardProp: propsChecker,
+})<{
+  $dataQualityTestRunStatus: DataQualityTestRunStatusEnum;
```

Comment:
You can use a shorter naming here, for example:
```suggestion
  $testRunStatus: DataQualityTestRunStatusEnum;
```
We can understand property name from component context, and with the shorter name your code will look better.